### DISCUSSION TITLE
Dimensions on Pentair Acid Tank

### DISCUSSION THREAD

**Discussion by jamestalmage** [tagyoureit/nodejs-poolController#736] [REM] [answered]
Anyone know the footprint of the [Pentair Acid Tank](https://sunplay.com/products/pentair-acid-tank-with-tank-mounted-pump-522472)? Can't find it anywhere.

**Comment by johnny2678** [comment, marked as answer]
> Anyone know the footprint of the [Pentair Acid Tank](https://sunplay.com/products/pentair-acid-tank-with-tank-mounted-pump-522472)? Can't find it anywhere.

I just happened to be out at the pad this AM... 13.5" x 13.5"
